room/forms.py

```diff
@@ -2,22 +2,32 @@
 
 from .models import Room
 
+
 class AddRoom(forms.ModelForm):
-    title = forms.CharField(label='Name of Your New Room', max_length=255)
+    title = forms.CharField(label="Name of Your New Room", max_length=255)
 
     class Meta:
         model = Room
-        fields = {'title',}
+        fields = {
+            "title",
+        }
+
```

Comment:
I think there's no particularly good reason to split across several lines. If an expression easily fits on one line, it's nearly always correct to leave it in 'compact' form.

Reply:
Same as above. Black working its magic.